polar-core/src/vm.rs

```diff
@@ -3934,4 +4023,35 @@ mod tests {
             QueryEvent::Done { result: true }
         ]);
     }
+
+    #[test]
+    fn test_log_level_should_print_for_level() {
+        let trace = LogLevel::Trace;
+        let info = LogLevel::Info;
+        let debug = LogLevel::Debug;
```

Comment:
nit
```suggestion
        use LogLevel::*;
```
and then reference them as `Trace` instead of `trace`, etc.